src/main/flight/mixer.c

```diff
@@ -425,6 +425,51 @@ static void updateDynLpfCutoffs(timeUs_t currentTimeUs, float throttle)
 }
 #endif
 
+static void applyMixerAdjustmentLinear(float *motorMix, bool airmodeEnabled) {
+    float motorMixNormalizationFactor = motorMixRange > 1.0f ? motorMixRange : 1.0f;
+    float motorMixDelta = 0.5f * motorMixRange;
```

Comment:
maybe
**const** float motorMixNormalizationFactor ....
**const** float motorMixDelta ...

Reply:
Might be good to define more const definitions for non mutables. I will go through refactoring once more, when it is finished